plugin/mode_indicator/mode_indicator.py

```diff
@@ -180,10 +185,19 @@ def on_update_settings(updated_settings: set[str]):
         update_indicator()
 
 
+def poll_microphone():
+    global current_microphone
```

Comment:
```suggestion
    # Ideally, we would have a callback instead of needing to poll. https://github.com/talonvoice/talon/issues/624
    global current_microphone
```